BothamNetworking/NSHTTPClient.swift

```diff
@@ -28,14 +30,16 @@ public class NSHTTPClient: HTTPClient {
     private func mapHTTPRequestToNSURLRequest(httpRequest: HTTPRequest) -> NSURLRequest {
         let components = NSURLComponents(string: httpRequest.url)
         if let params = httpRequest.parameters {
-            components?.queryItems = params.map {
-                NSURLQueryItem(name: $0.0, value: $0.1)
+            components?.queryItems = params.map { (key, value) in
+                let paramValue: String? = !value.isEmpty ? value : nil
```

Comment:
I thinks it's better to not do this mapping("" -> nil), and just set the value.
And keep this for post 1.0.0